.changes/next-release/feature-AWSSDKforJavav2-44b1dae.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "type": "feature", 
+    "description": "maven-compiler-plugin 3.7.0"
```

Comment:
sorry to nit-pick but can we get more descriptive descriptions? These go in the https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md so I'd prefer it was a full sentence: "Upgraded maven-compile-plugin from 3.6.0 -> 3.7.0"

If you had a whole list in a single commit then we could potentially bullet them like we did for the "2.0.0-preview-6" release - but individually it'd be good to be a bit more long-handed!